sdk/python/foundation-models/system/inference/text-to-image/safe-image-text-to-image-batch-deployment.ipynb

```diff
@@ -0,0 +1,786 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create an Azure AI Content Safety (AACS) enabled image to image batch endpoint (Preview)\n",
+    "### This notebook is under preview.\n",
+    "\n",
+    "### Steps to create an __AACS__ enabled __image to image__ batch endpoint\n",
+    "1. Create a __AACS__ enabled image-to-image batch endpoint with a custom [score_batch.py](./aacs-scoring-files/score/score_batch.py) script. This will integrate the batch endpoint with the AACS resource to moderate the response from the __image-to-image__ model and the request from the user.\n",
+    "2. Create a new __AACS__ enabled __image-to-image__ batch endpoint with a custom [score_batch.py](./aacs-scoring-files/score/score_batch.py) which will integrate with the __AACS__ resource to moderate the response from the __image-to-image__ model and the request from the user. To make the custom [score_batch.py](./aacs-scoring-files/score/score_batch.py) to successfully authenticated to the __AACS__ resource, use __Environment variable__ to pass the access key of the __AACS__ resource to the custom [score_batch.py](./aacs-scoring-files/score/score_batch.py) via environment variable. The custom [score_batch.py](./aacs-scoring-files/score/score_batch.py) can use the key directly to access the AACS resource. This option is less secure, if someone in your org has access to the endpoint, he/she can get the access key from the environment variable and use it to access the AACS resource.\n",
+    "\n",
+    "### Task\n",
+    "`image to image` task takes an original image, a text prompt as input. The model generates an image by modifying the original image.\n",
+    "\n",
+    " \n",
+    "### Model\n",
+    "Models that can perform the `image to image` task are tagged with `image-text-to-image`. We will use the `stabilityai-stable-diffusion-xl-refiner-1-0` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name.\n",
```

Comment:
Task name. Same comment as above. 

The task should be image-to-image.
Have left the same comment on model asset PR - https://github.com/Azure/azureml-assets/pull/1785.